packages/scripts/README.md

```diff
@@ -532,10 +532,11 @@ Should there be any situation where you want to provide your own webpack config,
 
 To extend the provided webpack config, or replace subsections within the provided webpack config, you can provide your own `webpack.config.js` file, `require` the provided `webpack.config.js` file, and use the [`spread` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to import all of or part of the provided configuration.
 
-In the example below, a `webpack.config.js` file is added to the root folder extending the provided webpack config to include [`css-loader`](https://github.com/webpack-contrib/css-loader) and [`style-loader`](https://github.com/webpack-contrib/style-loader):
+In the example below, a `webpack.config.js` file is added to the root folder extending the provided webpack config to include custom logic to parse module's source and convert it to a JavaScript object using [`toml`](https://www.npmjs.com/package/toml). It may be useful to import toml or other non-JSON files as JSON, without specific loaders:
```

Comment:
Is the idea with this change that we now build-in CSS support, so we need a different example of extending? If so, shouldn't we also have something here in the documentation which describes the new CSS support?

Reply:
Yes, this example won’t work anymore so I found something else. 

Yes, I mentioned that documentation is missing in https://github.com/WordPress/gutenberg/pull/21730#issuecomment-621326806.